index.js

```diff
@@ -0,0 +1,36 @@
+function addToList( item, listItems=[] ) {
+	// Don't add empty items or items that are already on the list (if we pass a list to check against...)
+	if ( item.length <= 0 || listItems.includes(item) ) {
+		/* maybe do something like display a message: item is already on the list */
+		return;
+	}
```

Comment:
Maybe split these 2 conditions into separate if statements since you'd want separate messages depending which is true (once/if it's implemented)

Reply:
done